apps/src/templates/courseOverview/AssignToSection.js

```diff
@@ -154,7 +159,8 @@ class AssignCourse extends Component {
         {dropdownOpen && (
           <div style={styles.dropdown}>
             <a
-              href={`${window.dashboard.CODE_ORG_URL}/teacher-dashboard?newSection=${courseId}#/sections`}
+              href={`${window.dashboard.CODE_ORG_URL}/teacher-dashboard?` +
+                queryString.stringify({courseId, scriptId}) + "#/sections"}
```

Comment:
Wait... `window.dashboard.CODE_ORG_URL` was available this whole time?  Should I be using this instead of building pegasusUrl helpers?